### PR TITLE
JS pre OOP

### PR DESCRIPTION
**Pull request by tavor118sn**
### JS PRE OOP

[Demo](https://tavor118sn.github.io/a-tiny-JS-world/)
[Code base](https://github.com/tavor118sn/a-tiny-JS-world)

The code is submitted in a dedicated feature branch.
Please, review.

### CODE REVIEW

File: submissions/tavor118sn/task_pre_oop/index.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+const inhabitants = [man, woman, direwolf, cat, catWoman];
+
+const makeMessage = (obj) => {
+  let friends = '';
+  if (obj.hasOwnProperty(friends) && obj.friends.length > 0) {
+    friends = obj.friends.join(', ');
+  }
+
+  return [
+    `${obj.species}`,
```
Comment (OleksiyRudenko) [inline submissions/tavor118sn/task_pre_oop/index.js:61]: Please explain `` `${obj.species}` ``

Comment (tavor118sn) [reply submissions/tavor118sn/task_pre_oop/index.js:61]: Not sure I understand the question. I need value of this key. I get it using dot notation.

Comment (tavor118sn) [reply submissions/tavor118sn/task_pre_oop/index.js:61]: Or do you mean backquotes are redundant in this case?

Comment (OleksiyRudenko) [reply submissions/tavor118sn/task_pre_oop/index.js:61]: Why did you use backticks?

Comment (tavor118sn) [reply submissions/tavor118sn/task_pre_oop/index.js:61]: 2 reasons:
1) to make each line look similar 
2) it is easier to add some formatting/tags in the future
But I can remove if they are redundant.

Comment (OleksiyRudenko) [reply submissions/tavor118sn/task_pre_oop/index.js:61]: Clear, thank you. That's what I wanted to hear.

**Review by OleksiyRudenko — APPROVED** [review]
@tavor118sn good job!

A side note:
This task requires pre-mature optimizations for students to have some experience with optimization on a small scale. In general, pre-mature optimizations should be avoided.

**Pull request merged**